1-Basics/Lesson-3.js

```diff
@@ -0,0 +1,9 @@
+// Lesson-3 Statements
+
+// Можно сделать так
+console.log('Mother of Dragons.');
+console.log('Dracarys!');
+
+// Но так будет более правильно и красиво
+// \n переносит значение на следующую строку
+console.log('\nRobert\nStannis\nRenly')
```

Comment:
There is another way to store (and print) multiline strings - just use backticks `string`;
Like this:
```javascript
const text = `Robert
              Stannis
              Renly`;
```